srcpkgs/v2ray/INSTALL.msg

```diff
@@ -0,0 +1,2 @@
+Consult /usr/share/doc/v2ray/README.voidlinux for additional configuration
+instructions.
```

Comment:
Please don't add noise install messages, this is not required and results in users just ignoring them all together.